modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -4417,15 +4417,26 @@ version 1.3.2.
 - [LPS-84119]: Update the [Liferay Gradle Plugins] dependency to version
 3.12.116.
 
-## 5.5.25 - 2018-08-29
+## 5.5.25 - 2018-08-28
 
 ### Changed
-- [LPS-84887]: Update the `scm.connection` property's default value to
-`scm:git:git@github.com:liferay/liferay-portal.git`.
-- [LPS-84887]: Update the `scm.developerConnection` property's default value to
-`scm:git:git@github.com:liferay/liferay-portal.git`.
-- [LPS-84887]: Update the `scm.url` property's default value to
-`http://github.com/liferay/liferay-portal`.
+- [LPS-84094]: Allow properties defined in `gradle-ext.properties` to be
+overridden by values defined in a `gradle-ext.properties` file
```

Comment:
@petershin should the first file stated here be `gradle.properties` instead of `gradle-ext.properties`?

Reply:
@codyhoag yes, that's correct!